.github/workflows/acctest.yml

```
@@ -35,48 +35,116 @@ concurrency:
           go-version-file: "go.mod"
-          check-latest: true
+          check-latest: false
 
       - name: Set up Terraform
         uses: "hashicorp/setup-terraform@v3"
+
+      # Pre-pull the controller image so a transient Docker Hub hiccup (e.g. a
+      # 429) doesn't fail the test step, and the cold pull is out of the test
+      # timeout budget. testcontainers reuses the already-present image.
+      - name: Pull UniFi controller image
+        if: ${{ !matrix.unifi_download_url }}
```

actionlint (1.7.12)

[error] property "unifi_download_url" is not defined in object type {unifi_version: any}

(expression)